### PR TITLE
Images can be attached to posts.

### PR DESCRIPTION
**Pull request by jcairo**

### CODE REVIEW

**Comment by mraypold** [comment]
:+1:

File: author_api/serializers.py [hunk 1]
```diff
@@ -9,6 +12,22 @@
...
 
+class ImageSerializer(serializers.BaseSerializer):
+    def to_representation(self, data):
+        return data.name
+
+    def to_internal_value(self, data):
+        try:
+            extension =  guess_extension(guess_type(data[0:23])[0])
+
+            if extension:
```
Comment (derekdowling) [inline author_api/serializers.py:23]: What happens if there is no extension? Does it throw an exception? If so the "if" is redundant. Either that or we're going to end up in a state where nothing gets returned.

File: author_api/serializers.py [hunk 2]
```diff
@@ -9,6 +12,22 @@
     FriendRelationship,
     FriendRequest )
 
+class ImageSerializer(serializers.BaseSerializer):
+    def to_representation(self, data):
+        return data.name
```
Comment (derekdowling) [inline author_api/serializers.py:17]: So data.name is the url in which it can be fetched from?

File: author_api/serializers.py (hunk 1 above)
Comment (jcairo) [inline author_api/serializers.py:23]: Your right. I'll update.

**Pull request merged**